### ISSUE TITLE
append area size

### ISSUE THREAD

**Issue by BenWilliams-NOAA** [PeteHulson-NOAA/SWO-simulation#1]
Seems from this line down is pretty much just appending the area size to the comp but otherwise the same as the previous functino - is that accurate? 
https://github.com/PeteHulson-NOAA/SWO-simulation/blob/a1ea0d8bd1195a9a09c1d69f1ad2f75af8707ff8/est_SzPop_strata.R#L102

**Comment by PeteHulson-NOAA** [comment]
Yes, it's basically just reverting a step or two and formatting in a way to spit out the strata level estimates rather than sum up for management area ests
